smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -225,6 +232,46 @@ private String getCommandExample(String serviceName, String configName, String c
                 + String.format("@see {@link %s} for command's `input` shape.%n", commandInput)
                 + String.format("@see {@link %s} for command's `response` shape.%n", commandOutput)
                 + String.format("@see {@link %s | config} for %s's `config` shape.%n", configName, serviceName);
+
+        return exampleDoc;
+    }
+
+    /**
+     * Handwritten examples from the operation ExamplesTrait.
+     */
+    private String getCuratedExamples(String commandName) {
+        String exampleDoc = "";
+        if (operation.getTrait(ExamplesTrait.class).isPresent()) {
+            List<ExamplesTrait.Example> examples = operation.getTrait(ExamplesTrait.class).get().getExamples();
+            StringBuilder buffer = new StringBuilder();
+
+            for (ExamplesTrait.Example example : examples) {
+                ObjectNode input = example.getInput();
+                Optional<ObjectNode> output = example.getOutput();
+                buffer
+                    .append("\n")
+                    .append(String.format("@example %s%n", example.getTitle()))
+                    .append("```javascript\n")
+                    .append(String.format("/* %s */%n", example.getDocumentation().orElse("")))
```

Comment:
Use single line comment?
```suggestion
                    .append(String.format("// %s%n", example.getDocumentation().orElse("")))
```

Reply:
This should be okay since we use single line comments currently and haven't run into any multi-line descriptions